docs/system-design.md

```
@@ -0,0 +1,228 @@
+    UserToken {
+        int Id PK
+        string Email
+        string Token
+        enum TokenType
+        datetime ExpiryDate
+        bool IsUsed
+    }
+
+```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)